src/assets/fabricator/styles/components/layout.css

```diff
@@ -0,0 +1,67 @@
+.f-Layout {
+  height: 100%;
+}
+
+.f-Layout.f-is-active {
```

Comment:
@tylersticka That last thing you said nailed it for me: clarity. It's easy to say "Every single style in fabricator is name-spaced. Period."